tools/rapids-logger

```diff
@@ -20,4 +20,4 @@ function rapids-logger {
   echo -e "\033[32m└${bar}┘\033[0m\n"
 }
 
-rapids-logger "$@"
+rapids-logger "$@" >&2
```

Comment:
This change is necessary to ensure that the command's logging output is not captured by variables when using `rapids-retry` like this:

```sh
SOME_VAR=$(rapids-retry ...)
```